app/models/scanned_resource.rb

```diff
@@ -12,11 +12,18 @@ class ScannedResource < Resource
   attribute :workflow_note, Valkyrie::Types::Array.of(WorkflowNote).optional
   attribute :file_metadata, Valkyrie::Types::Set.of(FileMetadata.optional)
   attribute :change_set, Valkyrie::Types::String
+  attribute :auth_token, Valkyrie::Types::String
 
   def self.can_have_manifests?
     true
   end
 
+  # Ensure that resources of this class cannot be accessed with an access token
```

Comment:
Can you double check this comment? To me tokenized_access? being true would seem to imply that it *can* be accessed with a token.

Reply:
I'm sorry for that oversight, this has been corrected.